README.rst

```diff
@@ -82,6 +82,21 @@ On a Debian based system, the installation is as follows::
     make && make install
     cd ..
 
+Installing the PostgresSQL session_exec extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+    git clone https://github.com/okbob/session_exec
```

Comment:
My guess is that production is the only one using 9.3 at this point. Maybe I'm hinting at something. ;-)